addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -343,6 +343,7 @@ $o-we-zindex: $o-we-overlay-zindex + 1 !default;
         }
 
         .o_we_colorpicker_switch_panel button {
+            @extend %btn;
```

Comment:
Seems like instead, all these `.o_we_colorpicker_switch_panel button` should just have had the `btn` class. So I'd expect them to be added in `snippets.xml` instead.